README.md

```diff
@@ -12,16 +48,17 @@ Running the application requires the following tools to be installed in your env
   * [npm](https://www.npmjs.com/)
   * [Docker](https://www.docker.com)
 
-### Running the application
-
-You need to create a slack bot and export the following values in your shell:
+You need to create a Slack App as required in the steps above and export the following values in your shell:
```

Comment:
I think this section should be restructured, we just want people to use the .env file I think for developing, there's a number of variables related to jira that aren't mentioned below that are in the .env file

Reply:
Please check now for updates.